menu/menu.yaml

```diff
@@ -875,6 +875,10 @@
       Articles:
       - Url: transports/upgrades/msmq-1to2
         Title: Version 1 to 2
+  - Title: Kafka
+    Articles:
+    - Url: transports/kafka
+      Title: Kafka transport
```

Comment:
As we discussed if we follow through on this PR, we shouldn't place this under transports as it sets the wrong expectations.

Reply:
There may be many that already have some sort of expectation, [based on other libraries](https://masstransit.io/documentation/transports/kafka).